transfuse-core/src/main/java/org/androidtransfuse/gen/FilerSourceCodeWriter.java

```diff
@@ -49,10 +50,8 @@ public OutputStream openBinary(JPackage jPackage, String fileName) throws IOExce
         //generate a source file based on package and fileName
         String qualified = toQualifiedClassName(jPackage, fileName);
         JavaFileObject sourceFile = filer.createSourceFile(qualified, originating.getOriginatingElements(qualified));
-
-        OutputStream os = sourceFile.openOutputStream();
+        OutputStream os = new WriterOutputStream(sourceFile.openWriter(), "UTF-8");
```

Comment:
Pass in `StandardCharsets.UTF_8` instead of the string?

Reply:
ah yes, i just replace it